_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,318 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that for any text content that auto-updates in a 10 minute time span, there are instruments to pause, stop, or hide it or to control its update frequency. The arbitrary 10 minute time span, selected so that testing this rule would not be impractical, is not included in WCAG. Content that updates less frequently may fail success criteria 2.2.2 without failing this rule.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] in the [flat tree][] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element in the [flat tree][] with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+## Expectation
+
+For each test target there is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] as the test target or can be found in a [clearly labeled location][] from that [web page][], to achieve one of the following objectives:
+
+- pause and resume the change of the [visible text content][]; or
+- stop the change of the [visible text content][]; or
```

Comment:
Or perhaps "stop the automated change of the visible text content". Presume this would be applicable to the pause/resume phrasing also.

Reply:
We don't use "automated" anywhere in the applicability or the expectation, so I don't think it's a good idea to introduce the term. Applicability specifies that the applicable changes are the ones that happen without user intervention (that's how we defined automated), and the changes the expectation addresses are those. Do you find this ambiguous?

Reply:
I think I was agreeing more with the intent of what Adil suggested, rather than the exact wording. Automation may not be in the applicability or expectation, but it is referenced in the description.

I'm aware of set-ups where sports results might update automatically with Javascript, but with that feature disabled a page refresh would still update the content. I'm not sure if an adjustment is necessary, but the thing being paused or stopped is the automation of a change/update rather than there being no means to update the text. I understand if you don't feel that clarification is needed.

Reply:
But we already have an assumption for that :-)

"Any content changes are enabled by the content of the HTML document the test target belongs to. Changes originating from any other sources (e.g. browser shortcuts, browser extensions, browser settings, user agents, external browser applications) are not considered."
If the change is caused by the user refreshing the page, it is not considered for this rule.

Reply:
If you think that is clear enough.